tests/GenFu.Tests/GenFuTests.cs

```diff
@@ -447,5 +447,43 @@ public void MethodCanBeSet()
             Assert.True(!string.IsNullOrEmpty(person.GetMiddleName()));
             Assert.Equal(expected, person.GetMiddleName());
         }
+
+        [Fact]
+        public void NullableDateWillGetRandomNullSeed()
+        {
+
+            var dates = new List<DateTime?>();
+            for(int x = 0; x<100; x++)
+            {
+                A.Reset();
+                var person = A.New(new Person());
+                dates.Add(person.DateOfDeath);
+            }
+
+            int nullItems = dates.Where(x => x == null).Count();
+            int notNullItems = dates.Where(x => x != null).Count();
+
+            // Should contain mix of Nulls and set values
+            Assert.True(nullItems > 0);
+            Assert.True(notNullItems > 0);
+        }
+
+        [Fact(Skip ="Not yet supported but should be")]
```

Comment:
IntIsPopulatedIfNullable() is skipped as this is currently not working. I have only done the code for <DateTime?>.
I noticed there was a convention of developers having "Skipped" tests as a way to review required features. If the community likes the work pattern I have done I can repeat this for other data types. For example <int?>. This test proved that, if nullable, the value is never set.